### ISSUE TITLE
Topbeat reporting inaccurate CPU

### ISSUE THREAD

**Issue by oazabir** [elastic/beats#2138]
TOPbeat reporting CPU use as 50,000,000,000,000% for some process.

![image](https://cloud.githubusercontent.com/assets/1611494/17278064/13ceae26-574b-11e6-92b1-dc8524f3a6b2.png)

Another example:

![image](https://cloud.githubusercontent.com/assets/1611494/17278068/2687447e-574b-11e6-9d66-0f072a3d08ad.png)

Here's /proc/cpuinfo last block:

```
processor	: 23
vendor_id	: GenuineIntel
cpu family	: 6
model		: 45
model name	: Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz
stepping	: 7
microcode	: 1808
cpu MHz		: 1999.926
cache size	: 15360 KB
physical id	: 1
siblings	: 12
core id		: 5
cpu cores	: 6
apicid		: 43
initial apicid	: 43
fpu		: yes
fpu_exception	: yes
cpuid level	: 13
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good xtopology nonstop_tsc aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer aes xsave avx lahf_lm ida arat epb xsaveopt pln pts dts tpr_shadow vnmi flexpriority ept vpid
bogomips	: 3999.44
clflush size	: 64
cache_alignment	: 64
address sizes	: 46 bits physical, 48 bits virtual
power management:
```

And output from top:

```
top - 20:20:07 up 20 days,  7:48,  2 users,  load average: 3.61, 3.40, 3.83
Tasks: 1915 total,   7 running, 1866 sleeping,   0 stopped,  42 zombie
Cpu(s): 21.1%us,  2.5%sy,  0.0%ni, 76.4%id,  0.0%wa,  0.0%hi,  0.1%si,  0.0%st
Mem:  132124840k total, 130690640k used,  1434200k free,   362216k buffers
Swap:  2097148k total,   853384k used,  1243764k free, 101799848k cached

  PID USER      PR  NI  VIRT  RES  SHR S %CPU %MEM    TIME+  COMMAND                                                                                                                                      
 5795 mysql     20   0 45.7g  10g 5440 S 121.5  8.3 796:23.39 mysqld                                                                                                                                       
24372 islamico  20   0  467m  88m  62m S 58.3  0.1   0:01.78 php                                                                                                                                           
24352 islamico  20   0     0    0    0 Z 46.2  0.0   0:01.82 php <defunct>                                                                                                                                 
24390 islamico  20   0  458m  67m  50m R 32.7  0.1   0:01.00 php                                                                                                                                           
24377 islamico  20   0     0    0    0 Z 23.2  0.0   0:00.71 php <defunct>                                                                                                                                 
24396 islamico  20   0  451m  34m  25m R 22.6  0.0   0:00.69 php                                                                                                                                           
24392 islamico  20   0     0    0    0 Z 19.3  0.0   0:00.59 php <defunct>   
```

**Comment by oazabir** [comment]
Topbeat installed via yum.
topbeat.x86_64                                                                                        1.2.3-1

**Comment by andrewkroh** [comment]
I believe this issue has been fixed in 1.3 branch (not released yet). See https://github.com/elastic/beats/pull/1928. Can you try the [snapshot build](https://download.elastic.co/beats/topbeat/topbeat-1.3.0-SNAPSHOT-x86_64.rpm) to see if it fixes this issue?
